app/javascript/packs/actions/answers.js

```diff
@@ -6,13 +6,15 @@ import {
   ANSWER_FETCH_IN_PROGRESS,
   ANSWER_FETCH_SUCCESS,
   ANSWER_FETCH_FAILURE,
+  UPDATE_LOCAL_ANSWER,
   REMOVE_ANSWER,
 } from '../constants';
 import { post, patch } from '../fetch/requester';
 
-export function answerFetchInProgress(buildingId, answer) {
+function answerFetchInProgress(buildingId, answer) {
   return {
     type: ANSWER_FETCH_IN_PROGRESS,
+    // TODO: no longer need fetch status because we have separate actions for each stage
```

Comment:
See comment in `reducers/answers.js` -- this `status` property is what I mean by "fetch constants."